plugins/inputs/gtfs/README.md

```diff
@@ -0,0 +1,81 @@
+# GTFS
+
+This input plugin can be used to gather vehicle positions, trip updates and
+service alerts from a GTFS-realtime feed. [GTFS-realtime](https://developers.google.com/transit/gtfs-realtime) 
+is an extension of the [General Transit Feed Specification](https://developers.google.com/transit/gtfs).
+
+### Configuration
+
+This plugin can be configured to collect any combination of vehicle positions, trip updates
+and service alerts. Enable any one of these three input types by populating its URL in the
+plugin configuration. At least one of the three must be configured.
+
+```toml
+[[inputs.gtfs]]
+  ## API Key
+  # key = "${GTFS_API_KEY}"
+  ## URL for fetching vehicle positions
+  # vehicle_positions_url = "https://host.test/VehiclePositions.pb"
+  ## URL for fetching vehicle positions
+  # trip_updates_url = "https://host.test/TripUpdates.pb"
+  ## URL for fetching vehicle positions
+  # service_alerts_url = "https://host.test/ServiceAlerts.pb"
+
+  ## Optional HTTP Basic Auth Credentials
+  # username = "username"
+  # password = "pa$$word"
+
+  ## Optional TLS Config
+  # tls_ca = "/etc/telegraf/ca.pem"
+  # tls_cert = "/etc/telegraf/cert.pem"
+  # tls_key = "/etc/telegraf/key.pem"
+  ## Use TLS but skip chain & host verification
+  # insecure_skip_verify = false
+
+  ## Amount of time allowed to complete the HTTP request
+  # timeout = "5s"
+```
+
+### Metrics
+
+Metrics can be collected for each of three GTFS-realtime feed types: vehicle positions, trip updates and service alerts.
+
+#### Vehicle Positions
+
+The vehicle `position` measurement captures information about the vehicles including location and congestion level.
+
+##### Fields 
+
+##### Tags
+
+#### Trip Updates
+
+The trip `update` measurement captures information about delays, cancellations and changed routes.
+
+##### Fields 
+
+##### Tags
+
+#### Service Alerts
+
+The service `alert` measurement captures information about stop moves and unforeseen events 
+affecting a station, route or the entire network.
+
+##### Fields 
+
+##### Tags
+
+### Examples
+
+An example of vehicle positions from the Boston's MBTA GTFS-realtime feed:
```

Comment:
Is there a public URL for trying this out? If so, please add here so users can check what to expect from the plugin.

Reply:
There is no public URL without an API key, but I can provide more details on how to get an example setup working against the Boston feed, including links to acquiring an API key (its a quick form request).

Reply:
It would be helpful for people playing with this to get a jump-start. Maybe a link to a tutorial would suffice.